crypto/cms/cms_lib.c

```diff
@@ -104,6 +104,7 @@ BIO *CMS_dataInit(CMS_ContentInfo *cms, BIO *icont)
 
 }
 
+/* unfortunately cannot constify SMIME_write_ASN1() due to this funcion */
```

Comment:
typo 'funcion'

Reply:
Pleased to hear you like this work.

I've fixed this typo (and made sure there is no further one like this).